admin/tests/acceptance/authenticated/users/get-test.js

```diff
@@ -24,10 +24,14 @@ module('Acceptance | authenticated/users/get', function (hooks) {
   test('displays user detail information page', async function (assert) {
     // given
     this.intl = this.owner.lookup('service:intl');
-    const user = await _buildAndAuthenticateUser(this.server, { email: 'john.harry@example.net', username: null });
+    const user = await _buildAndAuthenticateUser(this.server, {
+      email: 'john.harry@example.net',
+      username: 'john.harry0101',
+    });
     const expectedOrganizationMembershipsCount = 2;
     const expectedParticipationCount = 1;
     const expectedCertificationCenterCount = 3;
+    const expectedConnexionMethodsCount = 3;
```

Comment:
```suggestion
    const expectedAuthenticationMethodCount = 3;
```

Et faire les modifs correspondantes dans l'ensemble de ce fichier.